app/resources/bestowment_resource.rb

```diff
@@ -0,0 +1,5 @@
+class BestowmentResource < BaseResource
+  attributes :badge_id, :rank, :progress, :bestowed_at
+
+  has_one :user
```

Comment:
This is probably better to structure as another `Badge` resource (with no route of its own) which is `has_one`'d off of `Bestowment`, so we can access information on it